### PR TITLE
FEAT: add health check via middleware

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #10

### CODE REVIEW

**Comment by stijnmoreels** [comment]
Not sure though if this middleware should be removed when we have a full health endpoint?

File: src/Arcus.Template.WebApi/Startup.cs [hunk 1]
```diff
@@ -22,6 +26,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             #warning "Please configure application with authentication mechanism: https://webapi.arcus-azure.net/features/security/auth/shared-access-key"
 
             app.UseMvc();
+
+            app.UseHealthChecks("/health");
```
Comment (tomkerkhove) [inline src/Arcus.Template.WebApi/Startup.cs]: We don't really need this as we'll expose the endpoint ourselves based on the health middleware. See https://www.codit.eu/blog/documenting-asp-net-core-health-checks-with-openapi/ for more info

File: src/Arcus.Template.WebApi/Startup.cs [hunk 2]
```diff
@@ -11,6 +12,9 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+
+            services.AddHealthChecks()
+                    .AddCheck("self", () => HealthCheckResult.Healthy());
```
Comment (tomkerkhove) [inline src/Arcus.Template.WebApi/Startup.cs]: What happens if we leave this one out?

**Comment by tomkerkhove** [comment]
@stijnmoreels 	We will keep using the middleware, but not expose it via the built-in capabilities. See  codit.eu/blog/documenting-asp-net-core-health-checks-with-openapi  why

File: src/Arcus.Template.WebApi/Startup.cs (hunk 2 above)
Comment (fgheysels) [reply src/Arcus.Template.WebApi/Startup.cs]: I don't think you need the 'self' check.

**Pull request merged**